commands/up.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Alex Ellis 2017. All rights reserved.
```

Comment:
Please use `OpenFaaS Author(s) 2018` for copyright. One of the feedback from @alexellis  on my previous PR :)

Reply:
+1 for this

Reply:
yup, its a WIP pr was going to address this later - funnily enough I was looking for that header text when raising the PR and noticed this - https://github.com/openfaas/faas-cli/blob/master/commands/update_gitignore.go, you'll need to update that one as well :)